src/routes/test/+page.svelte

```diff
@@ -0,0 +1,9 @@
+<script>
+	import { enhance } from "$app/forms";
+	import { InputChip } from "@skeletonlabs/skeleton";
+</script>
+
+<form method="post" use:enhance>
+    <InputChip name="test" required />
+    <button>Submit</button>
```

Comment:
It shouldn't be a common use case, but did you test it with nested forms as well?

Reply:
Not yet, will do 👍